src/com/google/javascript/jscomp/CompilationLevel.java

```diff
@@ -163,7 +163,7 @@ private static void applyFullCompilationOptions(CompilerOptions options) {
     options.setRemoveAbstractMethods(true);
     options.setRemoveSuperMethods(true);
     options.setReserveRawExports(true);
-    options.setRenamingPolicy(VariableRenamingPolicy.ALL, PropertyRenamingPolicy.ALL_UNQUOTED);
+    options.setRenamingPolicy(VariableRenamingPolicy.ALL, PropertyRenamingPolicy.OFF);
```

Comment:
Ah, just realized I made a PR out of this already. It cannot be this simple, of course. Just wanted to document my local changes for now.